src/go/rpk/pkg/cli/topic/describe.go

```diff
@@ -31,20 +32,32 @@ func newDescribeCommand(fs afero.Fs, p *config.Params) *cobra.Command {
 		summary    bool
 		configs    bool
 		partitions bool
+		re         bool
 		stable     bool
 	)
 	cmd := &cobra.Command{
-		Use:     "describe [TOPIC]",
+		Use:     "describe [TOPICS]",
 		Aliases: []string{"info"},
-		Short:   "Describe a topic",
-		Long: `Describe a topic.
+		Short:   "Describe topics",
+		Long: `Describe topics.
 
-This command prints detailed information about a topic. There are three
+This command prints detailed information about topics. There are three
 potential sections: a summary of the topic, the topic configs, and a detailed
 partitions section. By default, the summary and configs sections are printed.
```

Comment:
Is it just a list/array of partitions? What does this section output look like?

Reply:
Thank you for looking @kbatuigas. Here's the default output

```
SUMMARY
=======
NAME        foo
PARTITIONS  1
REPLICAS    1

CONFIGS
=======
KEY                                   VALUE       SOURCE
cleanup.policy                        delete      DEFAULT_CONFIG
compression.type                      producer    DEFAULT_CONFIG
---- snip -----
segment.bytes                         134217728   DEFAULT_CONFIG
segment.ms                            1209600000  DEFAULT_CONFIG
```

The partition section looks like this:

```
PARTITIONS
==========
PARTITION  LEADER  EPOCH  REPLICAS  LOG-START-OFFSET  HIGH-WATERMARK
0          0       6      [0]       0                 10
1          0       1      [0]       0                 0
2          0       1      [0]       0                 0
3          0       1      [0]       0                 0
```